.storybook/pages/FeedbackOverview/FeedbackOverview.tsx

```diff
@@ -51,7 +51,6 @@ export const FeedbackOverview: React.FC = () => (
               <div className="fpo u-margin-bottom-xl">Progress bar</div>
               <div className="fpo u-margin-bottom-lg">Filters</div>
               <Table
-                behavior="stacked"
```

Comment:
I'm thinking of using `stacked` here but re-styling this to match closer to this. I know Sean and I were discussing potentially changing up their comps with this a bit: https://www.figma.com/file/hDfTdVzAAuf2DDb3hVSnFM/Notebooks%3A-Feedback-Overview?node-id=257%3A36353

Right now we have a card treatment for this. Let's change it to only an underline for that. The actual data with the table and all of that can be done after this gets merged in

Reply:
@csnizik Stacked is what I added for the small screen responsive "card" layout. We may want to use this for the layout in that Figma file that is more of `bottom-bordered` "table row" on small screens. So if you look at that Figma file it's just restyling these small screen styles to just have a bottom border for now to look more like that